pkg/config/constants.go

```diff
@@ -1,3 +1,5 @@
+//revive:disable
```

Comment:
Just curious for my own sake (no need to leave a comment here)... why are we disabling the linting for this file? Did it have to do with the regex?